Sources/PackageDescription/Target.swift

```diff
@@ -147,7 +150,7 @@ public final class Target {
     }
     private var _linkerSettings: [LinkerSetting]?
 
-    /// The binary target's checksum.
```

Comment:
I think it would be better if we kept this more generic, it could easily go out of sync if we add support for more formats. I think "ZIP file that contains the referenced binary artifact" or even more generic "archive file that..." would be preferable.